packages/safe-kit/src/lib/sendAndDeployTransaction.ts

```diff
@@ -0,0 +1,37 @@
+import { TransactionBase, TransactionOptions } from '@safe-global/safe-core-sdk-types'
+import { SafeClientTransactionResult } from '../types'
+import { SafeAccountClient } from '../SafeAccountClient'
+import { AbstractSigner } from 'ethers'
+
+/**
+ * Sends a transaction using the SafeAccountClient client.
+ *
+ * @param {TransactionBase[]} transactions - An array of transactions to be sent.
+ * @param {TransactionOptions} options - Options for executing the transaction.
+ * @param {SafeAccountClient} safeClient - The SafeAccountClient client used to send the transaction.
+ * @returns {Promise<SafeClientTransactionResult>} A promise that resolves to the result of the transaction.
+ */
+export const sendAndDeployTransaction = async (
```

Comment:
This function looks very similar to `sendTransaction`. Have you considered merging them into one function with something like a `deploySafe` parameter flag?

Reply:
It's not that similar, only the creation and signing part. However, if I create a deploy() function, I should either send the transaction signed or perform the signing internally, so it will be not super clear the implementation inside. Apart from that i would like to keep all these function under the same interface `(tx, options, safeClient) => txResponse`. I will keep it like this for now and let's see how we can evolve them